x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -18772,8 +18770,6 @@
     "xpack.lens.indexPattern.fieldNoOperation": "Le champ {field} ne peut pas être utilisé sans opération",
     "xpack.lens.indexPattern.fieldsNotFound": "{count, plural, one {Champ} other {Champs}} {missingFields} {count, plural, one {introuvable} other {introuvables}}.",
     "xpack.lens.indexPattern.fieldsWrongType": "{count, plural, one {Champ} other {Champs}} {invalidFields} {count, plural,  other {}}de type incorrect",
-    "xpack.lens.indexPattern.filters.queryPlaceholderKql": "{example}",
-    "xpack.lens.indexPattern.filters.queryPlaceholderLucene": "{example}",
```

Comment:
Can we simply rename these with the new scope? Since that's all that's changing.

Reply:
The team confirmed that deleting these does not increase translation overhead.